StudioCore/MsbEditor/Universe.cs

```diff
@@ -752,15 +752,15 @@ public async void LoadMapAsync(string mapid)
             task = job.StartJobAsync();
             tasks.Add(task);
 
-            if (FeatureFlags.LoadDS3Navmeshes)
+            if (FeatureFlags.LoadNavmeshes)
```

Comment:
You don't need 3 feature flags here. Just loadDS3Navmeshes and LoadNavmeshes. I'd go on to rename them LoadLegacyNavmehses and LoadHavokNavmeshes as that's what they differentiate.